Dockerfile

```diff
@@ -72,14 +72,27 @@ RUN mkdir -p ./vendor-assets && \
     find public/ -regex "public\/${regex_path}.*\.${regex_files}" -exec cp --parent "{}" vendor-assets/  \;
 
 
+###
+
+
+FROM node:20 AS assets-build
+
+WORKDIR /code/public/app/themes/justice
+COPY ./public/app/themes/justice ./
+
+RUN npm ci
+RUN npm run production
+RUN rm -rf node_modules
+
 
 ###
 
 
 FROM base-fpm AS build-fpm
 
 WORKDIR /var/www/html
-COPY --from=build-fpm-composer --chown=www-data:www-data /var/www/html /var/www/html
+COPY --from=build-fpm-composer --chown=www-data:www-data /var/www/html .
+COPY --from=assets-build       --chown=www-data:www-data /code/public/app/themes/justice/dist/php ./public/app/themes/justice/dist/php
```

Comment:
Then this would be:

```
COPY --from=assets-build --chown=www-data:www-data /code/dist/php ./public/app/themes/justice/dist/php
```

Reply:
Thanks, I've done this in the new commit. Except the folder name is node, to match the folder name in docker-compose.yml